### PR TITLE
View event page

### PR DESCRIPTION
**Pull request by silverbabyyyy**
Refactored view event page, made it prettier than ever before.

### CODE REVIEW

File: src/app/components/event-page/event-page.component.html [hunk 1]
```diff
@@ -1,84 +1,96 @@
...
-            <div class="event-information-first-row">
-                <div  class="event-text-information">
-                    <div class="title-date">
-                        <div class="event-title">{{event.title}}</div>
-                        <div class="start-date">{{ event.startingDate | date:'short' }}</div>
+<div class="event">
+    <mat-card class="event-card">
+        <div class="top-div">
+            <div class="event-image">
+                <img  [src]="event.backgroundImage != 'string' ? event.backgroundImage : 'https://media.istockphoto.com/photos/baby-husky-puppy-playing-in-the-park-picture-id989949432?k=20&m=989949432&s=170667a&w=0&h=SWbImVzYQZnWUerppitlWdqDo3rAfaztstpbqiO2Pjo='" alt="">
```
Comment (meda-racasan) [inline src/app/components/event-page/event-page.component.html]: Do we really want the puppy image to show up on every event that doesn't have a picture? :D

File: src/app/components/event-page/event-page.component.html [hunk 2]
```diff
@@ -1,84 +1,96 @@
...
-                        <div class="required-participants">Required participants: {{event.minimumParticipants}}</div>
                     </div>
                 </div>
-                <div class="admin-buttons" *ngIf="isHostMode"}}>
-                    <button mat-button class="admin-buttons-style"><mat-icon>edit</mat-icon></button>
-                    <button mat-button class="admin-buttons-style" (click)="deleteEvent()"><mat-icon>clear</mat-icon></button>
+                <div>
+                    <div class="title-date">
+                        <!-- <h2 class="event-title">{{ event.title }}</h2> -->
+                        <h2 class="event-title">SOCIAL & ROLLERSKATING </h2>
```
Comment (meda-racasan) [inline src/app/components/event-page/event-page.component.html]: Title should be dynamic, not hardcoded.

File: src/app/components/event-page/event-page.component.html [hunk 3]
```diff
@@ -1,84 +1,96 @@
...
-            <div  class="participants-card" (click)="openParticipantsDialog()">
-                <mat-icon class="participants-icon">group</mat-icon>
-                <div class="bold-element">Participants: </div>
-                <div class="current-participants-number">{{event.maximumParticipants / 2 + 3}} / {{event.maximumParticipants}}</div>
+        <div class="middle-div">
+            <div class="default-message">Event will automatically cancel if minimum number of participants is not met by deadline</div>
+            <div class="participants-data">
+                <div class="participants-card" (click)="openParticipantsDialog()">
+                    <div class="bold-element">Participants: </div>
+                    <div class="current-participants-number">{{event.maximumParticipants / 2 + 3}} / {{event.maximumParticipants}}</div>
```
Comment (meda-racasan) [inline src/app/components/event-page/event-page.component.html]: Why do you need this formula? `event.maximumParticipants / 2 + 3`

File: src/app/components/event-page/event-page.component.html [hunk 4]
```diff
@@ -1,84 +1,96 @@
...
-                    <div class="bold-element">Location</div>
-                    <div class="event-location-details">
-                        <div>{{event.eventLocation.street}} {{event.eventLocation.address}}</div>
-                        <div>{{event.eventLocation.city}}, {{event.eventLocation.province}}</div>
-                        <div><a href={{getMapLink()}}>View Map</a></div>
+        <div class="bottom-div">
+            <div class="event-description">
+                <!-- <div class="event-descripton-text">{{event.description}}</div> -->
+                <div class="event-descripton-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</div>
```
Comment (meda-racasan) [inline src/app/components/event-page/event-page.component.html]: Don't hardcode the description

File: src/app/components/event-page/event-page.component.html [hunk 5]
```diff
@@ -1,84 +1,96 @@
...
+                </mat-chip-list>
+            </div>
+            <div class="event-details">
+                <div class="event-details-text">
+                    <div class="location-text">
+                        <h2 class="location">Location:</h2>
+                        <div class="event-location-details">
+                            <div>{{event.eventLocation.street}} {{event.eventLocation.address}}</div>
+                            <div>{{event.eventLocation.city}}, {{event.eventLocation.province}}</div>
+                            <div><a href={{getMapLink()}}>View Map</a></div>
```
Comment (meda-racasan) [inline src/app/components/event-page/event-page.component.html]: Don't use functions inside the template. Set the map link on a property on the object.

File: src/app/components/event-page/event-page.component.html [hunk 6]
```diff
@@ -1,84 +1,96 @@
...
-                    <div>{{event.endingDate | date:'short'}}</div>
-                </div>
-                <div>
-                    <div class="bold-element">Host</div>
-                    <div class="host-details">
-                        <div>Email: {{event.contactEmail}}</div>
-                        <div>Phone: {{event.contactPhone}}</div>
+                    <div>
+                        <h2 class="date">Date & Time:</h2>
+                        <div class="starting-date">{{event.startingDate | date:'dd.mm.yyyy, HH:ss'}} -</div>
```
Comment (meda-racasan) [inline src/app/components/event-page/event-page.component.html]: When you see something gets repetitive, create a constant for it :D. For example for the date format string.

**Comment by silverbabyyyy** [comment]
Have to fix number of participants and host buttons before merge.

**Pull request closed without merging**